### ISSUE TITLE
how is dropout disabled while testing?

### ISSUE THREAD

**Issue by ianni67** [despoisj/DeepAudioClassification#9]
This may be a silly question, depending on how skilled is the reader.
How is dropout disabled for testing? I did not find any instruction/parameter/whatever aimed at that in the code. I suppose it should be automatic in TFLearn or Tensorflow, but could not find that in the documentation. 
Any kind soul willing to shed some light?

**Comment by despoisj** [comment]
Exactly, this is implicitely deactivated by the backend as dropout is only used for regularization and should therefore not occur at testing time. 

Julien

**Comment by ianni67** [comment]
Thank you very much for your prompt help. 
I'm having some hard time in understanding why I get dancing accuracy values when repeating train/test cycles over and over with the same parameters and deleting the checkpoints at each cycle.
I get different accuracies at each cycle, varying by abt 5%. It looks a bit too much, but I'm still learning...
